include/crm/crm.h

```diff
@@ -140,7 +140,7 @@ extern char *crm_system_name;
 #  define CRM_OP_QUIT		"quit"
 #  define CRM_OP_LOCAL_SHUTDOWN 	"start_shutdown"
 #  define CRM_OP_SHUTDOWN_REQ	"req_shutdown"
-#  define CRM_OP_SHUTDOWN 	"do_shutdown"
+#  define CRM_OP_SHUTDOWN   PCMK_ACTION_DO_SHUTDOWN
```

Comment:
Can we deprecate `CRM_OP_SHUTDOWN` and/or get rid of the code that checks for it in the controller? I could be overlooking it but I don't see anything in Pacemaker going back to 1.1.11 that sends a `CRM_OP_SHUTDOWN` -- only code that checks for an incoming one.

Nothing in current sbd or dlm either.

Reply:
The DC forwards the graph action as a controller command to the relevant peer. In that context it's a CPG command name rather than an action name per se. Eventually it should be renamed `PCMK__CONTROLD_CMD_SHUTDOWN` or similar.